<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JSONClass</class>
 <widget class="QMainWindow" name="JSONClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JSON</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>文本：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>图像化</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>图形：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTreeView" name="treeView">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="headerHidden">
       <bool>true</bool>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>703</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文本操作</string>
    </property>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
    <addaction name="action_2"/>
    <addaction name="action_8"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
    <addaction name="action"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFuzhi">
   <property name="text">
    <string>fuzhi</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>清空</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>格式化</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>压缩</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="json.qrc"/>
 </resources>
 <connections/>
</ui>
